CryptKeeperZK/crypt-keeper-extension
# feat: backup onboarding flow 😈

## Explanation

This PR adds support for uploading backup file as a part of onboarding flow.

Details are below:
- [x] Add backup flow for new users
- [x] Fix styles for onboarding page
- [x] Fix styles for backup upload page
- [x] Support redirect param for common paths
- [x] Integrate new user backup upload
- [x] Update backup upload method to load data in the right sequence

## More Information

Blocked by #664
Related to #330 

## Screenshots/Screencaps

![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/4923cdf3-53ed-4b4d-a848-70ccf49eeca2)

![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/f89f77e3-b7d0-4fd2-8c4b-a0c92875544b)


## Manual Testing Steps

1. Install the extension
2. Create new account
3. Create some identities, generate some proofs from demo
4. Download backup file
5. Delete and install the extension again
6. Try to upload backup file from onboarding flow
7. Check the accounts, identities and all data related

## Pre-Merge Checklist


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
